src/Net/Http/HttpOptions.php

```diff
@@ -18,6 +24,14 @@ public function __construct()
     }
 
     /**
+     * @return SslOptions
+     */
+    public function getSslOptions()
+    {
+        return $this->sslOptions ?: $this->sslOptions = new SslOptions;
```

Comment:
Not really sure abut this one, you expect the getter to give you the current value of the internal property, but in this case the property is null and the getter returns an object, maybe another name?